### DISCUSSION TITLE
OneDrive 获取token一直失败

### DISCUSSION THREAD

**Discussion by haiwan-xm** [AlistGo/alist#5525] [General]
OneDrive 获取token一直失败
因为更换文件目录没备份数据导致存储数据丢失
现在重新配置OD存储报错了 一直无法刷新获取token
再登录页面后直接报错
登录
抱歉，登录时遇到问题。

AADSTS50011: The redirect URI 'https://alist.nn.ci/tool/onedrive/callback' specified in the request does not match the redirect URIs configured for the application '2*****06-9269-4f43-b***-057*******04'. Make sure the redirect URI sent in the request matches one added to your application in the Azure portal. Navigate to https://aka.ms/redirectUriMismatchError to learn more about how to fix this.

**Comment by XIAOCHEN-NYE** [comment]
我和你的问题一样，目前也不知道如何解决
